tasks/au/data.py

```diff
@@ -263,6 +216,65 @@ def _fetch_requirements(self):
         with (open('{}/{}'.format(dir_path, '{}_{}_requirements.json'.format(self.profile, self.year)))) as f:
             return json.load(f)
 
+    def _process_col(self, line, rows, session, column_reqs, cols, tablename, source, license, country, unittags, subsectiontags):
+        col_id = line[1]            # B: short
+        col_name = line[2]          # C: name
+        denominators = line[3]      # D: denominators
+        col_unit = line[5]          # F: unit
+
+        if tablename == '{}02'.format(self.profile[0]):
+            col_agg = line[8]       # I: AGG (for B02 only)
+        else:
+            col_agg = None
+        tabledesc = line[10]        # K: Table description
+
+        denominators = denominators.split('|')
+
+        targets_dict = {}
+        for denom_id in denominators:
+            denom_id = denom_id.strip()
+            if not denom_id:
+                continue
+
+            reltype = 'denominator'
+            if col_agg in ['median', 'average']:
+                reltype = 'universe'
+
+            if denom_id in column_reqs:
+                targets_dict[column_reqs[denom_id].get(session)] = reltype
+            else:
+                if denom_id not in cols:
+                    # we load denominators recursively to avoid having to order
+                    # them in the source CSV file
+                    self._process_col(rows[denom_id], rows, session, column_reqs, cols, tablename, source, license, country, unittags, subsectiontags)
+                targets_dict[cols[denom_id]] = reltype
+
+        targets_dict.pop(None, None)
+
+        cols[col_id] = OBSColumn(
+            id=col_id,
+            type='Numeric',
+            name=col_name,
+            description=tabledesc,
+            # Ranking of importance, sometimes used to favor certain measures in auto-selection
+            # Weight of 0 will hide this column from the user.  We generally use between 0 and 10
+            weight=5,
+            aggregate=col_agg or 'sum',
+            # Tags are our way of noting aspects of this measure like its unit, the country
+            # it's relevant to, and which section(s) of the catalog it should appear in
+            tags=[source, license, country, unittags[col_unit]],
+            targets=targets_dict
+        )
+
+        # append the rest of the subsection tags
```

Comment:
Can we do this before the OBSColumn creation and pass it in the constructor?

I'd expect something like:
`col_subsections = line[6].split('|')` to be at the top, where all other line parsing is done

`subsection_tags = [subsectiontags[subsection.strip()] for ss in col_subsections]`

`tags=[source, _license, country, unittags[col_unit]] + subsection_tags` in the constructor above

Just an idea, not sure if this is done after the constructor for any reason, but looks weird